### PR TITLE
Implemented json to console metric logging

### PR DESCRIPTION
**Pull request by tcoupland**

### CODE REVIEW

File: target/classes/META-INF/maven/kixi.metrics/kixi.metrics/pom.properties [hunk 1]
```diff
@@ -1,5 +0,0 @@
-#Leiningen
-#Thu Feb 23 17:05:43 GMT 2017
```
Comment (elisehuard) [inline target/classes/META-INF/maven/kixi.metrics/kixi.metrics/pom.properties:2]: should these be saved in version control?

File: target/stale/leiningen.core.classpath.extract-native-dependencies [hunk 2]
```diff
@@ -1 +0,0 @@
...d.metrics/metrics-jvm {:vsn "3.1.2", :native-prefix nil, :native? false}, com.fasterxml.jackson.dataformat/jackson-dataformat-cbor {:vsn "2.8.6", :native-prefix nil, :native? false}, com.fasterxml.jackson.core/jackson-core {:vsn "2.8.6", :native-prefix nil, :native? false}, cheshire {:vsn "5.7.0", :native-prefix nil, :native? false}, metrics-clojure {:vsn "2.9.0", :native-prefix nil, :native? false}, io.dropwizard.metrics/metrics-core {:vsn "3.1.2", :native-prefix nil, :native? false}, org.clojure/tools.nrepl {:vsn "0.2.12", :native-prefix nil, :native? false}, clojure-complete {:vsn "0.2.4", :native-prefix nil, :native? false}, cider/cider-nrepl {:vsn "0.15.0-20170207.071608-9", :native-prefix nil, :native? false}, org.tcrawley/dynapath {:vsn "0.2.5", :native-prefix nil, :native? false}, tigris {:vsn "0.1.1", :native-prefix nil, :native? false}, org.slf4j/slf4j-api {:vsn "1.7.7", :native-prefix nil, :native? false}, metrics-clojure-jvm {:vsn "2.9.0", :native-prefix nil, :native? false}, refactor-nrepl {:vsn "2.3.0-20170210.162113-13", :native-prefix nil, :native? false}, com.fasterxml.jackson.dataformat/jackson-dataformat-smile {:vsn "2.8.6", :native-prefix nil, :native? false}}}]
```
Comment (elisehuard) [inline target/stale/leiningen.core.classpath.extract-native-dependencies:1]: or this one?

**Review by elisehuard — COMMENTED** [review]
just questions about target files in there.  didn't know about all those types of metrics, nice!

**Review by acron0 — APPROVED** [review]

File: test/target/classes/META-INF/maven/kixi.metrics/kixi.metrics/pom.properties [hunk 3]
```diff
@@ -1,5 +0,0 @@
-#Leiningen
```
Comment (elisehuard) [inline test/target/classes/META-INF/maven/kixi.metrics/kixi.metrics/pom.properties:1]: I'll approve but these 2 should also be removed?

**Pull request merged**